sktime/utils/data_io.py

```diff
@@ -60,11 +55,6 @@ class values.
         all time-series and (if relevant) a column "class_vals" the
         associated class values.
     """
-    warn(
-        "This function has moved to datasets/_data_io, this version will be removed "
-        "in V0.10",
-        FutureWarning,
-    )
```

Comment:
Why are you undeprecating rather than using the on it suggests (as of 0.10) should be in datasets/_data_io

Reply:
thanks chris, I deleted the wrong deprecation warning!

Reply:
this is odd, I'm not sure what happened to the warnings, I'll put them back in